Modified pdf tables

TuomasBorman opened the pull request:
Hi, 

here is the implementation that uses `knitr::kable()`. html version has tables as they were in previous versions, but in pdf tables are scaled if they don't fit. 

There is still one weird thing going on: in page 16 those two tables are in wrong position.

Regarding to rendering the pdf version: I got the pdf rendering working when I cleared the environment completely, i.e., removed objects and restarted R

This pull request solves the same problem as https://github.com/microbiome/course_2021_radboud/pull/31, but differently. It might be worth to discuss, which implementation we keep

-Tuomas

TuomasBorman committed:
Modified pdf tables

ChouaibB commented:
Hi,
I tried to clear the variable env before rendering each book, however for some reason it still does not work.
Yet I checked the online book, and all figures are working, pdf download is working with knitr tables also appearing there.
For the margin across pages of the pdf, I solved it earlier by adding to the header of the index.Rmd the following and it worked:
```
classoption: oneside
geometry:
  - top=30mm
  - left=15mm
```

TuomasBorman committed:
Sync

Sync

# Conflicts:
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_asv_aitchison-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_asv_bc-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_coloring-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_genus-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_phylum_aitchison-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/permanova_coefs-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-14-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-15-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-18-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-20-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-22-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-25-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-29-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-33-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-37-1.pdf
#	docs/alpha-diversity.html
#	docs/beta-diversity.html
#	docs/differential-abundance-analysis.html
#	docs/getting-started.html
#	docs/importing-microbiome-data.html
#	docs/index.html
#	docs/microbiome-data-exploration.html
#	docs/miscellaneous-material.html
#	docs/program.html
#	docs/radboud2021_material.pdf
#	docs/search_index.json
#	docs/study-material.html

TuomasBorman committed:
Sync

Sync

# Conflicts:
#	docs/alpha-diversity.html
#	docs/beta-diversity.html
#	docs/differential-abundance-analysis.html
#	docs/main.R
#	docs/microbiome-data-exploration.html
#	docs/radboud2021_material.pdf
#	docs/radboud2021_material.tex
#	docs/search_index.json
#	main.R

TuomasBorman committed:
Sync

Sync

# Conflicts:
#	07-beta.Rmd
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_asv_aitchison-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_asv_bc-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_coloring-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_genus-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/pcoa_phylum_aitchison-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/permanova_coefs-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-14-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-15-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-18-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-20-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-22-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-25-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-29-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-33-1.pdf
#	_bookdown_files/radboud2021_material_files/figure-latex/unnamed-chunk-37-1.pdf
#	docs/beta-diversity.html
#	docs/radboud2021_material.pdf
#	docs/radboud2021_material.tex
#	docs/search_index.json
